src/EditorFeatures/VisualBasicTest/MakeFieldReadonly/MakeFieldReadonlyTests.vb

```diff
@@ -0,0 +1,670 @@
+"Class C
+    Private _goo As Integer = 0
+    Private ReadOnly _bar As Integer = 0
+    Private _fizz As Integer = 0
+End Class")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeFieldReadonly)>
+        Public Async Function MultipleFieldsAssignedInline_VBSpecialForms01() As Task
+            Await TestInRegularAndScriptAsync(
+"Class C
+    Dim [|x|] As Integer, y As String
+End Class",
+"Class C
+    Private ReadOnly x As Integer
+    Private y As String
+End Class")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeFieldReadonly)>
+        Public Async Function MultipleFieldsAssignedInline_VBSpecialForms02() As Task
+            Await TestInRegularAndScriptAsync(
+"Class C
+    Dim x As Integer, [|y|] As String
+End Class",
+"Class C
+    Private x As Integer
+    Private ReadOnly y As String
+End Class")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeFieldReadonly)>
+        Public Async Function MultipleFieldsAssignedInline_VBSpecialForms03() As Task
+            Await TestInRegularAndScriptAsync(
+"Class C
+    Dim [|x|], y As Integer, z, w As String
+End Class",
+"Class C
+    Private ReadOnly x As Integer
+    Private y As Integer
+    Private z As String
+    Private w As String
+End Class")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeFieldReadonly)>
+        Public Async Function MultipleFieldsAssignedInline_VBSpecialForms04() As Task
+            Await TestInRegularAndScriptAsync(
+"Class C
+    Dim x, [|y|] As Integer, z, w As String
+End Class",
+"Class C
+    Private x As Integer
+    Private ReadOnly y As Integer
+    Private z As String
+    Private w As String
+End Class")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeFieldReadonly)>
+        Public Async Function MultipleFieldsAssignedInline_VBSpecialForms05() As Task
+            Await TestInRegularAndScriptAsync(
+"Class C
+    Dim x, y As Integer, [|z|], w As String
+End Class",
+"Class C
+    Private x As Integer
+    Private y As Integer
+    Private ReadOnly z As String
+    Private w As String
+End Class")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeFieldReadonly)>
+        Public Async Function MultipleFieldsAssignedInline_VBSpecialForms06() As Task
+            Await TestInRegularAndScriptAsync(
+"Class C
+    Dim x, y As Integer, z, [|w|] As String
+End Class",
+"Class C
+    Private x As Integer
+    Private y As Integer
+    Private z As String
+    Private ReadOnly w As String
+End Class")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeFieldReadonly)>
+        Public Async Function MultipleFieldsAssignedInline_VBSpecialForms07() As Task
+            Await TestInRegularAndScriptAsync(
+"Class C
+    Dim [|x|], y() As Integer, z(), w As String
```

Comment:
Do we have a test for `As New` applicable to multiple declarators? For e.g., `Dim [|x|], y As New Integer()`, where `x` is candidate to become readonly should change to:
```
Dim ReadOnly x As New Integer()
Dim y As New Integer()
```

Reply:
Not yet, I'll get some added.